versions.yml

```diff
@@ -188,21 +209,52 @@ modGroups:
       desc: Adds Texture Features to entities in Optifine compatible format.
       url: https://cdn.modrinth.com/data/BVzZfTc1/versions/1Y3lxOX6/entity_texture_features_fabric_1.20.2-4.5.1.jar
 
+    - name: Continuity
+      license: LGPL-3
+      homepage: https://modrinth.com/mod/continuity
+      desc: Provides default connected textures
+      url: https://cdn.modrinth.com/data/1IjD5062/versions/JXhQlDZl/continuity-3.0.0-beta.4%2B1.20.2.jar
+      requires: [Indium]
+
   Camera Enhancements:
     - name: Better Third Person
       license: ARR
-      homepage: https://www.curseforge.com/minecraft/mc-mods/better-third-person
+      homepage: https://modrinth.com/mod/better-third-person
+      modrinthId: G1s2WpNo
       desc: Adds independent rotation of the camera in a third person view
-      url: https://mediafilez.forgecdn.net/files/4596/636/BetterThirdPerson-Fabric-1.20-1.9.0.jar
+      url: https://cdn.modrinth.com/data/G1s2WpNo/versions/1yJzEpzh/BetterThirdPerson-Fabric-1.20-1.9.0.jar
 
     - name: Shoulder Surfing Reloaded
       license: MIT
-      homepage: https://www.curseforge.com/minecraft/mc-mods/shoulder-surfing-reloaded
+      homepage: https://modrinth.com/mod/shoulder-surfing-reloaded
+      modrinthId: kepjj2sy
       desc: Revamped over-the-shoulder F5 third-person camera view
-      url: https://mediafilez.forgecdn.net/files/4745/844/ShoulderSurfing-Fabric-1.20.1-2.5.jar
+      url: https://cdn.modrinth.com/data/kepjj2sy/versions/LHE2t99w/ShoulderSurfing-Fabric-1.20.1-2.5.jar
       requires: [Forge Config API Port]
 
-  # Audio Enhancements:
+    - name: Custom FOV
+      license: LGPL 3.0 or later
+      homepage: https://modrinth.com/mod/custom-fov-illusive
+      desc: Allows players to customize various field of view settings
+      url: https://cdn.modrinth.com/data/uZAcFHLd/versions/y3ZRuxm1/customfov-fabric-6.2.1%2B1.20.1.jar
+
+    - name: Camera Overhaul
+      license: MIT
+      homepage: https://modrinth.com/mod/cameraoverhaul
+      modrinthId: m0oRwcZx
+      desc: Adds various camera tilting
+      url: https://cdn.modrinth.com/data/m0oRwcZx/versions/fzCKxnmb/CameraOverhaul-1.4.0-fabric-universal.jar
+      configUrl: https://github.com/Wynntils/launchy-config/releases/download/v0.0.1-rc.0/CameraOverhaul.zip
+      requires: [Cloth Config]
+
+  Audio Enhancements:
+    - name: Drip Sounds
+      license: LGPL-3
+      homepage: https://modrinth.com/mod/dripsounds-fabric
+      modrinthId: T8MMXTpr
+      desc: Adds sounds for drip particles landing
+      url: https://cdn.modrinth.com/data/T8MMXTpr/versions/7GB1hLrr/DripSounds-1.19.4-0.3.2.jar
```

Comment:
Is this confirmed to work on 1.20.2?

Reply:
I tested every mod I updated here. Did some quick lootruns, caves and grinded some xp. I only updated mods that caused no issues at all.